lact-daemon/src/server/handler.rs

```diff
@@ -932,6 +933,21 @@ fn load_controllers(
         }
     });
 
+    let libdrm_amdgpu = if sysfs_only {
+        None
+    } else {
+        match LibDrmAmdgpu::new() {
+            Ok(libdrm_amdgpu) => {
+                info!("libdrm and libdrm_amdgpu initialized");
+                Some(Rc::new(libdrm_amdgpu))
```

Comment:
I don't believe this needs to be wrapped in an `Rc`, as `LibDrmAmdgpu` is a cloneable struct with `Arc`s internally: https://github.com/Umio-Yasuno/libdrm-amdgpu-sys-rs/blob/main/lib.rs#L75

Reply:
Done.  

https://github.com/ilya-zlobintsev/LACT/pull/438/commits/af7aecca4c0dd792e89afc8c536078afab77a072